### PR TITLE
Complete view of an individual article

### PR DESCRIPTION
**Pull request by BitKoda**
View an individual article. Article displays on a new page with a URL  /articles/:article_id

### CODE REVIEW

File: src/components/ArticlesList.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ const ArticlesList = () => {
     if(isLoading) return <p>Loading...</p>
     
     return (
-        <>
+        <section className="section__cards">
         {articles.map((article) =>  {
             console.log(article)
```
Comment (daviidbartlett) [inline src/components/ArticlesList.jsx]: remove console.log

File: src/utils/api.js [hunk 2]
```diff
@@ -5,10 +5,42 @@ const newslyApi = axios.create({
...
+
+// export const getTopics = () => {
+//     return newslyApi.get("/topics").then((res) => {
+//         return res.data.topics;
+//     })
+// }
+
+// export const addTopic = (form) => {
+//     return newslyApi
+//     .post("/topics", form)
```
Comment (daviidbartlett) [inline src/utils/api.js]: why are all of these in here? Make sure you working through the project in vertical slices.

File: src/pages/ArticlePage.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                title={article.title}
+                topic={article.topic}
+                author={article.author}
+                body={article.body}
+                created_at={formatDate(article.created_at)}
+                votes={article.votes}
+                comment_count={article.comment_count}
+            />
+            <div id="article__comments">
+
```
Comment (daviidbartlett) [inline src/pages/ArticlePage.jsx]: phantom div?

File: src/components/ArticlesList.jsx (hunk 1 above)
Comment (BitKoda) [reply src/components/ArticlesList.jsx]: I noticed this just before our chat and removed it - thanks!

File: src/utils/api.js (hunk 2 above)
Comment (BitKoda) [reply src/utils/api.js]: Removed.

File: src/pages/ArticlePage.jsx (hunk 3 above)
Comment (BitKoda) [reply src/pages/ArticlePage.jsx]: Placeholder-- removed.

**Pull request merged**